### PR TITLE
4.3: migrate traditional (some minor cosmetic edits)

### PR DESCRIPTION
**Pull request by keichwa**
# Some hints

Consider adding an entry to the `CHANGELOG.md` file in the toplevel directory.
Cosmetic changes such as fixing typos do not need log entries (nevertheless it is important to fix typos, etc.)!
In the `manager-4.3`, the hidden `.changelog` file with a leading dot is still in use.

Add Description, Target branches, and related Links below the following section titles.

In the Description, just enter a summary of why you created this PR (and, if available, add any relevant diagram).

# Description

Short summary of why you created this PR (if you added documentation, please add any relevant diagram).

# Target branches

* Which product version this PR applies to (Uyuni, SUMA 4.3, SUMA MU X.Y.Z, or SUMA development version).  This information can be helpful if `ifeval` statements are needed to publish it for certain products only.
* Does this PR need to be backported? If yes, create an issue for tracking it and add the link to this PR.
* Whenever possible, cross-reference each backport PR here, so that all backports can be easily accessed from the description.

Backport targets (edit as needed):

- 4.3

# Links
- This PR tracks issue #<insert spacewalk issue, if any>
- Related development PR #<insert PR link, if any>

### CODE REVIEW

File: modules/client-configuration/pages/contact-methods-migrate-traditional.adoc [hunk 1]
```diff
@@ -10,26 +10,29 @@ This could be done by:
...
-[NOTE]
-====
-Use  the activation key created before to generate the bootstrap script.
-====
 
-. Once the bootstrap script is generated, it's stored on the server in the [path]``/srv/www/htdocs/pub/bootstrap`` directory.
-  Alternatively, you can access the bootstrap script over HTTPS.
-  Replace [literal]``<example.com>`` with the host name of your {productname} Server:
-+
+// FiXME: Where exactly did we create this activation key?
```
Comment (vandabarata) [inline modules/client-configuration/pages/contact-methods-migrate-traditional.adoc:27]: We're supposed to create it before bootstrapping the traditional client, so it should already exist when we get to this point - but I wasn't sure how to mention it

File: modules/client-configuration/pages/contact-methods-migrate-traditional.adoc [hunk 2]
```diff
@@ -10,26 +10,29 @@ This could be done by:
...
+//        Do we actually need this hint?
```
Comment (vandabarata) [inline modules/client-configuration/pages/contact-methods-migrate-traditional.adoc:28]: Yes, it's really important to use the activation key when creating the bootstrap script for this or it just won't work.

File: modules/client-configuration/pages/contact-methods-migrate-traditional.adoc (hunk 1 above)
Comment (keichwa) [reply modules/client-configuration/pages/contact-methods-migrate-traditional.adoc]: ok, thanks.  I hope I got it right now.

File: modules/client-configuration/pages/contact-methods-migrate-traditional.adoc (hunk 2 above)
Comment (keichwa) [reply modules/client-configuration/pages/contact-methods-migrate-traditional.adoc]: ok

**Review by 0rnela — APPROVED** [review]

**Review by jcayouette — APPROVED** [review]

**Pull request merged**